README.md

```diff
@@ -1,10 +1,62 @@
 # Police Data Accessibility Project Scrapers
+This repo contains the record scrapers (and associated tooling) to further the goals of the Police Data Accessibility Project. Thank you for your interest in contributing!
 
-This repo contains the record scrapers (and associated tooling) to further the goals of the Police Data Accessibility Project.
+# Getting Started
+## Quick start
+1. [Clone](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/cloning-a-repository) this repo.
+2. Make a copy of the template folder in the appropriate jurisdiction folder.
+3. Code your scraper.
+4. Scrape sample data from the source and add a truncated version to the folder so we understand the kind of data your scraper generates.
+5. Complete the readme to the best of your ability.
+6. If you know how to use Splunk, complete the config file.
+7. Submit a Pull Request for approval.
 
-## Structure
+## Legal
+Only scrapers that comply with our [legal guidelines](https://docs.google.com/document/d/1gjnH0S18iBI20K1pfs4M3wuMqcLE_ZSgt71ITUY2Fbk/edit) will be merged into this repo.
 
-There are going to be a lot of scrapers. Try to stick to the format of `$STATE/$COUNTY/$RECORD_TYPE`. If there are state-level records being scraped, something like `$STATE/state/$RECORD_TYPE` is fine. Please avoid dashes and spaces in directory names.
```

Comment:
Why are the guidelines for repo folder structure being removed?

Reply:
@zgoulson these can stay, but it's described in the section where templates are defined & part of the process is creating your section of the directory. I'm not super picky, because one of the first things we make should be a little meta-scraper to monitor for the presence of scrapers in directories. We should consider scaffolding out the entire file structure to preclude the need for people to do this correctly at all.